docs/schema/merging.md

```diff
@@ -18,7 +18,7 @@ Guidance: [Updates and deletions](../guidance/build/merging)
 
 ### Discarded fields
 
-In the release schema, `"omitWhenMerged": true` is declared on fields that must be discarded during merging. These are presently: `id`, `date` and `tag`.
+In the release schema, `"omitWhenMerged": true` is declared on fields that must be discarded during merging. These are presently: `id`, `date` `publisher` and `tag`.
 
 * For a compiled release:
   * Both the fields and their values are discarded, because they are metadata about the individual releases; the compiled release replaces these with its own metadata.
```

Comment:
```suggestion
  * Both the fields and their values are discarded, because they are metadata about the individual releases. The compiled release replaces these with its own metadata, as described below.
```